wordpress-mobile/WordPress-Android
# Issue/fixing jittery cursor during media upload

This is a client-side fix for [Aztec issue #394](https://github.com/wordpress-mobile/AztecEditor-Android/issues/394). ([PR](https://github.com/wordpress-mobile/AztecEditor-Android/pull/401))

Instead of refreshing whole editor when inserting media, when uploading progress changes or media upload fails, we look up related image span using it's attributes and reset it.

To test:

1. While Aztec is enabled, open Post Editor.
2. Type some text.
3. While the image is uploading, put cursor somewhere in the text or in the title.
4. Notice that cursor is not moving when upload progress changes or upload is finished or failed.

Waiting for beta release branch.
